functions/packages.bash

```diff
@@ -133,6 +133,7 @@ exim_setup() {
     if ! relayuser="$(whiptail --title "Enter your public service mail user" --inputbox "\\nEnter your mail username you use with the public service to relay all outgoing mail to $smarthost" 10 80 "$relayuser" 3>&1 1>&2 2>&3)"; then echo "CANCELED"; return 0; fi
     if ! relaypass="$(whiptail --title "Enter your public service mail password" --passwordbox "\\nEnter the password to use for mailuser ${relayuser} to relay mail via ${smarthost}" 9 80 "$relaypass" 3>&1 1>&2 2>&3)"; then echo "CANCELED"; return 0; fi
     if ! adminmail="$(whiptail --title "Enter your administration user's mail address" --inputbox "\\nEnter the address to send system reports to" 9 80 "$adminmail" 3>&1 1>&2 2>&3)"; then echo "CANCELED"; return 0; fi
+    for i in smarthost smartport relayuser relaypass adminmail; do store_in_conf $i; done
```

Comment:
Where is `store_in_conf` defined? I don't see it and don't recall it from prior code.

Reply:
openhabian@devpi:~/openhabian $ grep -n store_i */*
functions/openhabian.bash:73:##    store_in_conf(String parameter)
functions/openhabian.bash:75:store_in_conf() {